Tasks/MavenV3/task.json

```diff
@@ -17,7 +17,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 3,
-        "Minor": 158,
+        "Minor": 159,
         "Patch": 0
```

Comment:
In this case since `Minor` is already 158, please update `Patch` to 1

Reply:
Done !